### ISSUE TITLE
about Semantic3D

### ISSUE THREAD

**Issue by longmalongma** [loicland/superpoint_graph#178]
Hi, Which dataset in this　http://www.semantic3d.net/ need to be downloaded?Ｔhere are many files in it.

**Comment by loicland** [comment]
if you only want to try the pretrained network you can only download the reduced set. If you want to retrain you need to downlaod the training set + labels.

**Comment by longmalongma** [comment]
> if you only want to try the pretrained network you can only download the reduced set. If you want to retrain you need to downlaod the training set + labels.
thanks.
